klarna-on-demand/src/main/java/com/klarna/ondemand/Crypto.java

```diff
@@ -0,0 +1,12 @@
+package com.klarna.ondemand;
```

Comment:
It may be worth adding a small class doc here just to say that this class is internal and shouldn't be used directly.